src/Hl7.Fhir.Specification.Tests/Validation/BindingValidationTests.cs

```diff
@@ -110,8 +110,8 @@ public void TestCodingValidation()
 
             c = new Coding("http://snomed.info/sct", "160244002");
             result = val2.Validate(c.ToTypedElement(), vc);
-            Assert.True(result.Success);
-            Assert.NotEmpty(result.Where(type: OperationOutcome.IssueType.NotSupported));
+            Assert.False(result.Success);
```

Comment:
Is this correct? It changes from success to non-success for the same test values?

Reply:
That must cause regressions later in the chain, no - e.g. a different result for Simplifier than before?

Reply:
Yeah, I think it's correct like this. LocalTerminologyService can't validate the code -> false -> error in validator..